appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
XML Calabash comes with Saxon bundled. At the moment, Travis runs XProc with the bundled Saxon. See [these job logs](https://travis-ci.org/AirQuick/xspec/builds/207687594). They all indicate Saxon 9.7.0.11.
```
INFO : file:/home/travis/build/AirQuick/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc:41:44:Testing with SAXON HE 9.7.0.11
```

If we like to test XProc with another Saxon, we probably need to adjust classpath and/or test with XML Calabash 1.1.15-9**6**.